roles/sap_general_preconfigure/README.md

```diff
@@ -1,341 +1,74 @@
+<!-- BEGIN Title -->
 # sap_general_preconfigure Ansible Role
+<!-- END Title -->
+![Ansible Lint for sap_general_preconfigure](https://github.com/sap-linuxlab/community.sap_install/actions/workflows/ansible-lint-sap_general_preconfigure.yml/badge.svg)
 
-This role installs required packages and performs configuration steps which are required for installing and running SAP NetWeaver or SAP HANA. Specific installation and configuration steps on top of these basic steps are performed with roles sap-netweaver-preconfigure and sap-hana-preconfigure. Future implementations may reduce the scope of this role, for example if certain installation or configuration steps are done in the more specific roles.
+## Description
+<!-- BEGIN Description -->
+Ansible Role `sap_general_preconfigure` is used to ensure that Managed nodes are configured to host SAP systems according to SAP Notes.
```

Comment:
Use `managed node` or `managed nodes` (both words lowercase) for all README.md files, just like in the document https://docs.ansible.com/ansible/latest/network/getting_started/basic_concepts.html .